### ISSUE TITLE
hello_video compiled

### ISSUE THREAD

**Issue by magnip** [tofuSCHNITZEL/pi_video_looper#4]
any chance of a compiled bin? I’m having trouble installing script on pi zero. I originally followed this page https://learn.adafruit.com/raspberry-pi-video-looper/installation . I did a fresh install of stretch and have gone through your script but can’t get past make install.

**Comment by tofuSCHNITZEL** [comment]
yes I had also issues at first with hello_video but with this image:
Raspbian Stretch Lite
Minimal image based on Debian Stretch
Version: November 2018
Release date: 2018-11-13
Kernel version: 4.14
it works with my RPi 3B+

which image did you use the same image?

But I also thought about adding a compiled hello_video

**Comment by tofuSCHNITZEL** [comment]
I also ordered a Pi Zero now so I can test it myself.

I pushed a new master with hello_video.bin included in the assets folder, please try if you can run this on your pi zero

**Comment by magnip** [comment]
I’m uploading my looper image to google drive will be about a hour for 1.97gb.
Had to remove my WiFi credentials... also added will be the Wpa_supplicant to edit with your details & add a ssh file to place on the fat partition in case you want to work remotely.
You mentioned your version of pi so if it don’t work I’ll patiently wait for your pi zero to arrive.
The image has been tested for around 3 months now and has only froze once (due to one video clip)
With your modifications it will now be possible to have a 4gb card with a weeks worth of looping art.. I can’t wait..    if it’s within your time frame and you fancy it could you have a go at cleaning the boot with a possible png or video intro we can set to run (similar to what retro pi has done)
Also note this has been running on a very old 2gb card and I always pull the power rather then shut down to test corruption. So far no trouble and that’s without being set to read only.
Will give you a shout with link as soon as it’s uploaded.

**Comment by tofuSCHNITZEL** [comment]
> I’m uploading my looper image to google drive will be about a hour for 1.97gb.

I can probably only do something with your image when my pi zero is here.

> png or video intro we can set to run (similar to what retro pi has done

I can send you the code that I use for my pi installations. which remove any trace of the pi boot screen and add a png splashscreen instead
you also gave me an idea, the next feature I will implement is to make a videofile play only once. this could be used to play an "intro" after each time the player / raspberry is started - would this be helpful?

**Comment by magnip** [comment]
here you go. I use balenaetcher to flash to memory card windows or mac and it should boot unless there is a compatibility issue. (arm 6/7 problems)
https://drive.google.com/open?id=1vCztkVJJdUH96S1CVxN6y9saNzxZPEvK

**Comment by tofuSCHNITZEL** [comment]
> 
> 
> here you go. I use balenaetcher to flash to memory card windows or mac and it should boot unless there is a compatibility issue. (arm 6/7 problems)
> https://drive.google.com/open?id=1vCztkVJJdUH96S1CVxN6y9saNzxZPEvK

okay I downloaded them, you can delete them again if you want.
have you tried my hello_video.bin?

**Comment by magnip** [comment]
yes very helpful it is going to be a very polished image for people to install on a zero. the work you have already done is exceptional if not for the fact you have now simulated what html web loops do only without a browser. The significance in this is we can now have what looks like hi res gifs in video format.
 as mentioned before ive searched for years for a stand alone animated gif player and believe me unless your running high power, gifs take up computing and will become choppy with resalution  and file size issues. The pi zero wont have none of it even when set for the highest performance.

**Comment by magnip** [comment]
have you tried my hello_video.bin?
you may have noticed by now im not that clever in the programming.
I have no idea what to do with it. I can ssh into my looper image and edit and transfer files so if you mean replace the bin in adafruits setup then ill have a go.

**Comment by tofuSCHNITZEL** [comment]
yes you copy the hello_video.bin into /user/local/bin and then restart the videolooper with sudo supervisorctl restart video_looper and see if it plays something

**Comment by magnip** [comment]
Managed to do that now by accessing the partition through windows deleting the original bin and replaced with your hello_video.bin.   Do i replace the video_looper.ini with yours to?

**Comment by tofuSCHNITZEL** [comment]
Interesting, why would there be a hello_video bin file to begin with if the build failed? You don't need to use my ini file if the rest of the installation (the python files) are not from my repository.
Maybe the best thing would be to wait for my pi zero and then I can test if my version will work on there like it does on the regular rpi.

**Comment by magnip** [comment]
The build has not failed. I replaced the hello_video.bin you suggested with copy and paste on the root partition of the build that i uploaded for you. inserted the card and booedt.....  looper working as normal.
ive now noticed due to how fast you are working on this in 24hours you have to rename video file to set number of repeats. this was not the case yesterday so retracing footsteps causing me a few issues.
Don't worry ill get there. My noob problems are messing up your page with a issue you have solved and should now be closed.
thanks for all your help over the weekend take care my friend. magnip

**Comment by magnip** [comment]
Working on pi zero w  perfectly with raspberry stretch light. This build is exceptional.
Well done on some excellent mods and thanks for the extra support. Seemlessly looping each video to my set preferences 👍🏼👍🏼👍🏼👍🏼👍🏼  👌
